wp-content/themes/core/routes/unsupported_browser/css/unsupported-browser.pcss

```diff
@@ -114,6 +117,10 @@ a {
 		font-weight: var(--legacy-heading-font-weight);
 		margin: var(--legacy-heading-spacing);
 	}
+
+	a{
```

Comment:
Missing a space here `a {`. This might also need a hover/focus state?

Reply:
✅